README.md

```diff
@@ -364,6 +364,7 @@ Services to securely store your Docker images.
 * [Rancher](https://github.com/rancher/rancher) - An open source project that provides a complete platform for operating Docker in production by [@rancher][rancher].
 * [Serf](https://github.com/hashicorp/serf) - Service orchestration and management tool by [@hashicorp][hashicorp]
 * [Shipyard](https://github.com/shipyard/shipyard) - Composable Docker Management
+* [Swarmpit](http://swarmpit.io/) - Lightweight Docker Swarm orchestration. Swarmpit provides clean way to manage your Docker Swarm cluster with various handful features such Service management, Smart search, Shared access, Private registries and more to come soon.
```

Comment:
Another thing, no need for capital letter at the begining of each work.
```
Lightweight Docker Swarm orchestration. Swarmpit provides clean way to manage your Docker Swarm cluster with various handful features such service management, smart search, shared access, private registries.
```
I removed the "more to come soon" as you can come back and add them here when the features are ready :D